BCStudentSoftwareDevTeam/celts
# 792 refactor program events table

Did:
-  Created a new column in the Events table called program_id
- Updated the test in the database with the program Events that were already created in the new Event.programEvents
- Reset the database and the new tests appeared in the database
-  Looked for every instance of the programEvent in the environment and redirect it to Event.program_id
- Deleted the programEventes Table
-  Updated all queries that were joining or using the programEvents Table to just use the Event Table.
Test:
- Reset Database
- Check if the right program_Id is inserted in the right Event Id
- Create a new Event and then Create a new Recursive Events
- Request Transcript of the new Event
- Check the program_id of your own profiles by checking the events you have attended.
